job-descriptions/software-engineer-intern.md

```diff
@@ -0,0 +1,32 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer Internship (San Francisco)
+
+### About the company
+
+At Sourcegraph, we are building a better, smarter foundation for software development. The innovations of the future will all rely on code. By empowering software developers today, we can literally bring the future sooner.
+
+We have already built the world's best [code search](https://about.sourcegraph.com/docs/search/) that is used and loved by large and small engineering organizations across the world.
+[Sourcegraph](https://about.sourcegraph.com/product/server) and our [browser extensions](https://about.sourcegraph.com/product/browser) provide IDE-like code intelligence (e.g. hover tooltips, go to definition, find references, and much more) in your code host, in code review, and everywhere you read code.
+
+Read [our blog](https://about.sourcegraph.com/blog/) to see what more we have been working on recently and read [our master plan](https://sourcegraph.com/plan) to see where we are going!
+
+### About the team
+
+Our small team consists of talented, mature, collaborative, driven individuals who are attracted to the massive problem we are tackling. We work in an open environment that treats people in a first-class manner and provides them with ownership, responsibility, and autonomy.
+
+### About the role
```

Comment:
Can we add a few "Examples of projects that interns have worked on in the past"? I think that's the most powerful way to communicate the scope. E.g.:

> Example Software Engineer Intern projects from past years:
> - Build a browser extension from scratch that communicates with the Sourcegraph API to inject code intelligence on GitHub.com.
> - ...

Reply:
Yeah, I am happy to add examples. All of our previous interns predate me starting so I don't know what all the projects were. If you know can you share?

Reply:
Great question — @attfarhan is my description of yours above correct? And @ggilmore can you share yours? And what did the other interns (Julian? Joe? Shantanu?) do?

Reply:
Yup that's correct.

Julien, Rohan, and I all spent some time working on the Sourcegraph Desktop app, which was an electron app + editor extension combo. The electron app had the Sourcegraph web view, and would update as you changed positions in your editor.

Rohan also spent a week or so on a Sourcegraph search integration for Alfred, which was pretty cool.